app/views/icons/_amazon.html.erb

```diff
@@ -0,0 +1 @@
.11.25.16.43.16s.32-.05.43-.16c.1-.11.16-.25.16-.42 0-.17-.05-.31-.16-.42a.573.573 0 0 0-.43-.16Zm.29 1.85h-.58c-.13 0-.2.06-.2.18v4.26c0 .12.06.18.2.18h.58c.13 0 .19-.06.19-.18v-4.25c0-.12-.06-.18-.19-.18M6.72 12.11c-.11.19-.28.3-.47.3-.26 0-.42-.19-.42-.48 0-.57.52-.67 1.02-.67v.14c0 .26 0 .48-.12.71m.12-1.39c-.39.04-.91.08-1.28.23-.45.17-.75.61-.73 1.09 0 .69.45 1.04 1.02 1.04.49 0 .75-.11 1.12-.49.12.17.17.26.39.45.05.03.12.02.16-.02l.52-.44c.06-.04.05-.12 0-.18-.12-.17-.26-.3-.26-.62v-1.04c0-.44.03-.84-.3-1.15-.26-.25-.7-.33-1.03-.33-.65 0-1.37.24-1.52 1.02-.02.08.04.12.1.14l.66.07c.06 0 .11-.06.12-.12.06-.27.29-.4.55-.4.14 0 .3.05.38.17.09.14.08.33.08.48v.08l.02.02Zm3.67 7.22c-.65.02-1.28.23-1.8.62-.2-.42-.58-.62-1.15-.62-.61.02-1.2.22-1.7.58l-.06-.28c-.01-.07-.03-.11-.07-.13a.265.265 0 0 0-.14-.03h-.4c-.12 0-.18.06-.18.18v4.13c0 .12.06.18.18.18h.54c.12 0 .18-.06.18-.18v-3.34c.4-.25.86-.39 1.33-.39.25 0 .43.07.54.21.11.14.17.36.17.66v2.87c0 .12.06.18.18.18h.54c.12 0 .18-.06.18-.18v-3.33c.4-.25.86-.39 1.34-.4.25 0 .43.07.54.21.12.14.18.36.18.66v2.87c0 .12.06.18.18.18h.53c.12 0 .18-.06.18-.18v-3.12c0-.42-.11-.75-.34-.98-.23-.23-.55-.34-.96-.34m6.17.16h-.53c-.12 0-.18.06-.18.18v3.31c-.44.28-.87.42-1.3.42-.28 0-.48-.07-.61-.23-.12-.15-.19-.38-.19-.7V18.3c0-.12-.06-.18-.18-.18h-.53c-.12 0-.18.06-.18.18v3.09c0 .43.11.77.34 1 .23.24.55.36.98.36.59 0 1.16-.21 1.72-.61l.06.31c0 .05.03.1.07.13.04.03.09.04.14.03h.39c.12 0 .18-.06.18-.18V18.3c0-.12-.06-.18-.18-.18m10.34 3.74s-.09.01-.14.03c-.3.08-.58.12-.83.12-.5 0-.87-.12-1.09-.38-.23-.26-.35-.66-.35-1.22v-.12c0-.57.11-.97.34-1.23.23-.26.6-.39 1.12-.39.26 0 .52.04.77.11.07.02.12.03.15.03.08 0 .12-.06.12-.18v-.25c0-.08-.01-.14-.04-.18a.307.307 0 0 0-.14-.09 2.97 2.97 0 0 0-.98-.17c-.73 0-1.29.21-1.69.63-.4.42-.6 1.02-.6 1.79s.19 1.37.58 1.78c.38.41.94.61 1.67.61.4 0 .75-.06 1.06-.19.07-.03.12-.06.15-.1.03-.05.04-.11.04-.17v-.24c0-.12-.04-.18-.12-.18M8.65 13.1h.69c.07 0 .13-.06.13-.12v-1.87c0-.41-.02-.97.48-.97s.42.58.42.97v1.87c0 .07.06.12.12.12h.69c.07 0 .13-.06.13-.12v-1.87c0-.2 0-.5.06-.67.07-.17.23-.28.42-.29.2 0 .36.07.41.31.03.14.02.51.02.66v1.87c0 .07.06.12.12.12h.69c.08 0 .14-.06.14-.12v-2.23c0-.38.04-.81-.18-1.11-.19-.25-.49-.39-.8-.39-.41 0-.79.21-.95.66-.19-.45-.47-.66-.9-.66-.43 0-.74.21-.91.66h-.02v-.46c0-.06-.06-.12-.12-.12h-.64c-.07 0-.13.06-.13.13V13c0 .06.06.11.12.12m7.17-1.72c0 .26 0 .48-.12.71-.11.19-.28.31-.48.31-.26 0-.42-.2-.42-.48 0-.57.52-.67 1.02-.67v.14-.01Zm0-.68c-.4.04-.91.08-1.28.23-.45.17-.75.61-.73 1.09 0 .69.45 1.04 1.02 1.04.49 0 .75-.11 1.12-.49.12.17.17.26.39.45.05.03.12.02.16-.02l.52-.44c.06-.04.05-.12 0-.18-.12-.17-.26-.3-.26-.62v-1.04c0-.44.03-.84-.3-1.15-.26-.25-.7-.33-1.03-.33-.65 0-1.37.24-1.52 1.02-.02.08.05.12.1.14l.66.07c.06 0 .11-.06.12-.12.06-.27.29-.4.55-.4.14 0 .3.05.38.17.09.14.08.33.08.48v.08l.02.02Zm-7.82 2.96a8.785 8.785 0 0 0 5.79 2.12c1.6 0 3.46-.48 4.74-1.39.21-.15.03-.37-.19-.29a11.892 11.892 0 0 1-10.2-.61c-.14-.08-.25.06-.13.17" /></g></g></svg>
```

Comment:
i put all of the `svg`s into their own `.erb` partials in order to render them. only thing of note is that i added a `width` and a `height` variable into each one, so that the size could be controlled and passed in.

Reply:
moved!